tools/flow_grid.py

```diff
@@ -0,0 +1,689 @@
+import numpy as np
+import pandas as pd
+import sys
+import ast
+
+class flow_grid(object):        
```

Comment:
I would actually like for the class name to make some reference to being a preprocessor.

Maybe something like `PreProc`, `GridPreProc`, or `ParamProc`. I don't know--I'm not great at coming up with names.